build/ci/phase-template.yml

```diff
@@ -34,9 +34,18 @@ phases:
       displayName: Download Benchmark Data
     - ${{ if eq(parameters.name, 'MacOS') }}:
       - script: brew update && brew install libomp mono-libgdiplus gettext && brew link gettext --force
-        displayName: Install runtime dependencies
+        displayName: Install runtime dependencies     
+    - script: $(_buildScript) -$(_configuration) -runtests -coverage=true
+      displayName: Run Tests with code coverage.
+      condition: and(eq(variables._phaseName, 'Windows_x64'), eq(variables._configuration, 'Debug'))
     - script: $(_buildScript) -$(_configuration) -runtests
-      displayName: Run Tests
+      displayName: Run Tests without code coverage.
+      condition: not(and(eq(variables._phaseName, 'Windows_x64'), eq(variables._configuration, 'Debug')))
+    - script: dotnet msbuild build/Codecov.proj /t:Codecov /p:UseCodecov=true
```

Comment:
This shouldn't be `dotnet msbuild` - which will use any old `dotnet` on the $PATH. Instead we should be using the SDK that we downloaded to the `Tools` directory. That way no matter what the build machine has installed on it, we always use a consistent SDK.